src/orange/detail/BIHEnclosingVolFinder.hh

```diff
@@ -3,26 +3,30 @@
 // See the top-level COPYRIGHT file for details.
 // SPDX-License-Identifier: (Apache-2.0 OR MIT)
 //---------------------------------------------------------------------------//
-//! \file orange/detail/BIHTraverser.hh
+//! \file orange/detail/BIHEnclosingVolFinder.hh
 //---------------------------------------------------------------------------//
 #pragma once
 
-#include "corecel/math/Algorithms.hh"
-
-#include "../BoundingBoxUtils.hh"
-#include "../OrangeData.hh"
+#include "BIHTraversalHelper.hh"
 
 namespace celeritas
 {
 namespace detail
 {
 //---------------------------------------------------------------------------//
 /*!
- * Traverse BIH tree using a depth-first search.
+ * Traverse the BIH tree to find a volume that contains a point.
  *
- * \todo move to top-level orange directory out of detail namespace
```

Comment:
This to-do still applies... I think the traverser should be in `univ` or the top level (or maybe we should try to reorganized based on runtime/configure time). It shouldn't be in `orange/detail` since it's used by ../univ/SimpleUnitTracker`

Reply:
Good point on SimpleUnitTracker. I will add the to-do back and we can remove things around as a separate MR.